maths/distance_formula.py

```diff
@@ -0,0 +1,15 @@
+def distance_formula(x1, y1, x2, y2):
+    """
+    Calculate the distance between the two coordinates (x1,y1) and (x2,y2).
+    >>> distance_formula(0, 0, 2, 2)
+    2.8284271247461903
+    >>> distance_formula(1, 0, 5, 2)
+    4.47213595499958
+    >>> distance_formula(0, 0, 5, 0)
+    5.0
+    """
+    return ((y2 - y1) ** 2 + (x2 - x1) ** 2) ** (1 / 2)
```

Comment:
Can you generalize this calculate the same for vectors of any dimensions instead of just 2-dimensional coordinates?
You can use numpy to implement it. 
Or you can call NumPy's inbuilt distance function. (which I don't recommend)

Reply:
I generalized it for any 2 vectors.